docs/EnvironmentVariables.md

```diff
@@ -0,0 +1,18 @@
+# Environment variables
+
+The PSGI server (`zonemaster_backend_rpcapi.psgi`) support the following
+environment variables.
+
+* `ZM_BACKEND_RPCAPI_LOGLEVEL`: Configure the log level, `trace` by default.
+  Accepted values are:
+  * `trace`
+  * `debug`
+  * `info` (also accepted: `inform`)
+  * `notice`
+  * `warning` (also accepted: `warn`)
+  * `error` (also accepted: `err`)
+  * `critical` (also accepted: `crit`, `fatal`)
+  * `alert`
+  * `emergency`
+* `ZM_BACKEND_RPCAPI_LOGJSON`: Setting it to any value will configure the
+  logger to log in JSON format, undefined by default.
```

Comment:
The emtpy string and the string `0` are also interpreted as false.

Reply:
Should be better now